domains/certificates/Query.API/API.IntegrationTests/Factories/RegistryConnectorApplicationFactory.cs

```diff
@@ -30,6 +30,9 @@ public class RegistryConnectorApplicationFactory : WebApplicationFactory<registr
     protected override void ConfigureWebHost(IWebHostBuilder builder)
     {
         builder.UseSetting("ConnectionStrings:Postgres", ConnectionString);
+        builder.UseSetting("Retry:DefaultFirstLevelRetryCount", "5");
```

Comment:
Why not use Options pattern here?

Reply:
Program uses Options pattern. I've tried using it here as well but found that it doesn't work always. This does